client/network/sync/src/lib.rs

```diff
@@ -233,6 +236,32 @@ impl Default for AllowedRequests {
 	}
 }
 
+struct SyncingMetrics {
+	pub import_queue_blocks_submitted: Counter<U64>,
+	pub import_queue_justifications_submitted: Counter<U64>,
+}
+
+impl SyncingMetrics {
+	fn register(registry: &Registry) -> Result<Self, PrometheusError> {
+		Ok(Self {
+			import_queue_blocks_submitted: register(
+				Counter::new(
+					"substrate_import_queue_blocks_submitted",
+					"Number of blocks submitted to the import queue.",
+				)?,
+				registry,
+			)?,
+			import_queue_justifications_submitted: register(
+				Counter::new(
+					"substrate_import_queue_justifications_submitted",
```

Comment:
```suggestion
					"substrate_sync_import_queue_justifications_submitted",
```

I know that you just copied this and the other name. However, they were already misleading. As someone else could also submit blocks/justifications to the import queue.